api/capi/src/nnstreamer-capi-single.c

```diff
@@ -382,9 +383,9 @@ ml_single_set_info_in_handle (ml_single_h single, gboolean is_input,
 
     if (tensors_info && !ml_tensors_info_is_equal (tensors_info, info)) {
       /* given input info is not matched with configured */
+      ml_tensors_info_destroy (info);
       if (is_input) {
         /* try to update tensors info */
-        ml_tensors_info_destroy (info);
         if (ml_single_update_info (single, tensors_info, &info) != ML_ERROR_NONE)
```

Comment:
mem-leak : ml_single_update_info() returns newly allocated 'info'. this should be freed

Reply:
done.